lib/_debug_agent.js

```diff
@@ -20,7 +20,14 @@ exports.start = function start() {
 
   agent.listen(process._debugAPI.port, function() {
     var addr = this.address();
-    process._rawDebug('Debugger listening on port %d', addr.port);
+    // SmartOS sometimes munges output from `process._rawDebug()`
+    // but Windows can't use `console.error()`.
+    // https://github.com/nodejs/node/issues/2476
+    if (process.platform == 'win32') {
```

Comment:
I usually feel like I need a bath after typing `==` instead of `===`. Not sure how that happened. Will change for sure.